tests/test_ldlf.py

```diff
@@ -82,7 +86,7 @@ def test_parser():
         LDLfEnd(),
     )
 
-    assert parser("!(<(!(A<->D))+((B;C)*)+(!last?)>[(true)*]end)") == LDLfNot(
+    assert parser("!(<(!(A<->D))+((B;C)*)+(?!last)>[(true)*]end)") == LDLfNot(
```

Comment:
I only moved the ? to the left of each expression. If we don't like this solution, i can restore all tests.

Reply:
that's OK